ts/setSingleTodoDeletionListener.ts

```diff
@@ -0,0 +1,23 @@
+import { storage } from "./storage.js";
+import { todosUl, todos } from "./todoapp.js";
+import { Todo } from "./types.js";
+
+function setSingleTodoDeletionListener() {
+  todosUl.addEventListener("click", (e) => {
```

Comment:
To be honest I don't understand this. This fn should register removal of one todo, but it listens to click of entire list